### DISCUSSION TITLE
The 2022 State of  Open Source Report

### DISCUSSION THREAD

**Discussion by wangleo61** [kaiyuanshe/ONES#58] [日常讨论]
李老师分享的由OSI和OpenLogic带来的关于Open Source的多维度的调查，有些还是比较有意思的
比如说OSPO组织的，每个大洲的比例都不太相同
[ebook-openlogic-the-2022-state-of-open-source-report.pdf](https://github.com/kaiyuanshe/ONES/files/8308432/ebook-openlogic-the-2022-state-of-open-source-report.pdf)

**Comment by will-ww** [comment]
> 李老师分享的由OSI和OpenLogic带来的关于Open Source的多维度的调查，有些还是比较有意思的 比如说OSPO组织的，每个大洲的比例都不太相同 [ebook-openlogic-the-2022-state-of-open-source-report.pdf](https://github.com/kaiyuanshe/ONES/files/8308432/ebook-openlogic-the-2022-state-of-open-source-report.pdf)

你的手太快了，正准备分享过来的~ :-)

**Reply by wangleo61** [reply]
哈哈哈，原来王老师周末也在呀
